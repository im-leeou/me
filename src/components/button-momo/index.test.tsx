import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';

import { ButtonMomo } from '.';

describe('components button momo', () => {
  it('renders correctly', () => {
    const result = render(<ButtonMomo>Click me</ButtonMomo>);

    expect(result).toMatchSnapshot();
  });

  it('renders as full width', () => {
    const result = render(<ButtonMomo fullW>Click me</ButtonMomo>);

    expect(result).toMatchSnapshot();
  });
});
