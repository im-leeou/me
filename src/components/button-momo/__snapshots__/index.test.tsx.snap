// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`components button momo > renders as full width 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <button
        class="center text-sm-rps transition-300 focus-shadow ring-border h-10 rounded-2 bg-[#D82D8B] px-4 py-2.5 text-white-500 ring-[#C5297E] hover:bg-[#C1177C] md:h-11"
      >
        <div
          class="mr-2 w-5 scale-[83.275%] md:w-6 md:scale-100"
        >
          <svg
            viewBox="0 0 24 24"
          >
            <symbol
              fill="none"
              id="0f406a85"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                clip-path="url(#clip0_2468_86)"
              >
                <path
                  d="M18.8803 1.5C16.0517 1.5 13.7597 3.78 13.7597 6.594C13.7597 9.40714 16.0526 11.6871 18.8803 11.6871C21.708 11.6871 24 9.40714 24 6.594C24 5.24229 23.46 3.94714 22.5 2.99143C21.5378 2.03565 20.2365 1.49947 18.8803 1.5ZM18.8803 8.75829C18.5953 8.75907 18.3129 8.70372 18.0493 8.59538C17.7857 8.48704 17.546 8.32784 17.3439 8.12687C17.1418 7.9259 16.9813 7.68709 16.8715 7.42408C16.7617 7.16107 16.7048 6.87901 16.704 6.594C16.7047 6.30892 16.7615 6.02677 16.8713 5.76366C16.981 5.50055 17.1415 5.26164 17.3436 5.06058C17.5457 4.85951 17.7854 4.70024 18.0491 4.59184C18.3128 4.48345 18.5952 4.42807 18.8803 4.42886C19.1653 4.42818 19.4476 4.48365 19.7112 4.59209C19.9748 4.70053 20.2144 4.85983 20.4164 5.06088C20.6184 5.26193 20.7789 5.50081 20.8885 5.76386C20.9982 6.02691 21.055 6.30899 21.0557 6.594C21.0555 6.8784 20.9991 7.15995 20.8899 7.42251C20.7806 7.68507 20.6205 7.92346 20.4189 8.124C20.01 8.53043 19.4568 8.75848 18.8803 8.75829ZM12.288 5.32029V11.6871H9.34371V5.28857C9.34371 4.98171 9.17914 4.69714 8.91171 4.54371C8.78026 4.46822 8.63131 4.42849 8.47971 4.42849C8.32812 4.42849 8.17917 4.46822 8.04771 4.54371C7.78114 4.698 7.61571 4.98086 7.61571 5.28857V11.6871H4.67229V5.28857C4.67229 4.81371 4.28486 4.42886 3.80829 4.42886C3.33086 4.42886 2.94429 4.81371 2.94429 5.28857V11.6871H0V5.32029C0 3.87343 0.822 2.55086 2.12314 1.90371C2.7612 1.58645 3.47468 1.45214 4.18443 1.51569C4.89418 1.57923 5.57245 1.83815 6.144 2.26371C6.71564 1.83798 7.39407 1.57897 8.10399 1.51542C8.81391 1.45188 9.52755 1.58628 10.1657 1.90371C10.8023 2.21935 11.3382 2.70635 11.7131 3.30993C12.0881 3.91352 12.2872 4.60974 12.288 5.32029ZM18.8794 13.8831C16.0526 13.8831 13.7597 16.1649 13.7597 18.978C13.7597 21.7911 16.0526 24.0711 18.8803 24.0711C21.708 24.0711 24 21.7911 24 18.978C24 17.6271 23.46 16.3311 22.5 15.3754C21.5378 14.4197 20.2365 13.8835 18.8803 13.884M18.8803 21.1423C18.4506 21.1429 18.0304 21.0164 17.6726 20.7786C17.3147 20.5408 17.0353 20.2024 16.8694 19.806C16.7048 19.4105 16.6617 18.9749 16.7457 18.5547C16.8297 18.1346 17.0369 17.749 17.3409 17.4471C17.646 17.144 18.0339 16.9377 18.4558 16.8542C18.8777 16.7706 19.3149 16.8135 19.7126 16.9774C20.1095 17.1403 20.449 17.4174 20.6882 17.7736C20.9273 18.1298 21.0552 18.549 21.0557 18.978C21.0541 19.5534 20.8241 20.1047 20.4161 20.5106C20.0082 20.9164 19.4557 21.1437 18.8803 21.1423ZM12.288 17.7043V24.0711H9.34371V17.6726C9.34371 17.3657 9.17914 17.0811 8.91171 16.9277C8.78026 16.8522 8.63131 16.8125 8.47971 16.8125C8.32812 16.8125 8.17917 16.8522 8.04771 16.9277C7.78114 17.082 7.61571 17.3649 7.61571 17.6726V24.0711H4.67229V17.6726C4.67229 17.1977 4.28486 16.8129 3.80829 16.8129C3.33086 16.8129 2.94429 17.1986 2.94429 17.6726V24.0711H0V17.7043C0 16.2574 0.822 14.934 2.12314 14.2869C2.76128 13.9697 3.4748 13.8355 4.18455 13.8992C4.8943 13.9629 5.57253 14.222 6.144 14.6477C6.71564 14.222 7.39407 13.963 8.10399 13.8994C8.81391 13.8359 9.52755 13.9703 10.1657 14.2877C10.8023 14.6034 11.3382 15.0904 11.7131 15.6939C12.0881 16.2975 12.2872 16.9937 12.288 17.7043Z"
                  fill="white"
                />
              </g>
              <defs>
                <clippath
                  id="clip0_2468_86"
                >
                  <rect
                    fill="white"
                    height="24"
                    transform="translate(0 0.5)"
                    width="24"
                  />
                </clippath>
              </defs>
            </symbol>
            <use
              href="#0f406a85"
            />
          </svg>
        </div>
        Momo me
      </button>
    </div>
    <div>
      <button
        class="center text-sm-rps transition-300 focus-shadow ring-border h-10 rounded-2 bg-[#D82D8B] px-4 py-2.5 text-white-500 ring-[#C5297E] hover:bg-[#C1177C] md:h-11 w-full"
      >
        <div
          class="mr-2 w-5 scale-[83.275%] md:w-6 md:scale-100"
        >
          <svg
            viewBox="0 0 24 24"
          >
            <symbol
              fill="none"
              id="0f406a85"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                clip-path="url(#clip0_2468_86)"
              >
                <path
                  d="M18.8803 1.5C16.0517 1.5 13.7597 3.78 13.7597 6.594C13.7597 9.40714 16.0526 11.6871 18.8803 11.6871C21.708 11.6871 24 9.40714 24 6.594C24 5.24229 23.46 3.94714 22.5 2.99143C21.5378 2.03565 20.2365 1.49947 18.8803 1.5ZM18.8803 8.75829C18.5953 8.75907 18.3129 8.70372 18.0493 8.59538C17.7857 8.48704 17.546 8.32784 17.3439 8.12687C17.1418 7.9259 16.9813 7.68709 16.8715 7.42408C16.7617 7.16107 16.7048 6.87901 16.704 6.594C16.7047 6.30892 16.7615 6.02677 16.8713 5.76366C16.981 5.50055 17.1415 5.26164 17.3436 5.06058C17.5457 4.85951 17.7854 4.70024 18.0491 4.59184C18.3128 4.48345 18.5952 4.42807 18.8803 4.42886C19.1653 4.42818 19.4476 4.48365 19.7112 4.59209C19.9748 4.70053 20.2144 4.85983 20.4164 5.06088C20.6184 5.26193 20.7789 5.50081 20.8885 5.76386C20.9982 6.02691 21.055 6.30899 21.0557 6.594C21.0555 6.8784 20.9991 7.15995 20.8899 7.42251C20.7806 7.68507 20.6205 7.92346 20.4189 8.124C20.01 8.53043 19.4568 8.75848 18.8803 8.75829ZM12.288 5.32029V11.6871H9.34371V5.28857C9.34371 4.98171 9.17914 4.69714 8.91171 4.54371C8.78026 4.46822 8.63131 4.42849 8.47971 4.42849C8.32812 4.42849 8.17917 4.46822 8.04771 4.54371C7.78114 4.698 7.61571 4.98086 7.61571 5.28857V11.6871H4.67229V5.28857C4.67229 4.81371 4.28486 4.42886 3.80829 4.42886C3.33086 4.42886 2.94429 4.81371 2.94429 5.28857V11.6871H0V5.32029C0 3.87343 0.822 2.55086 2.12314 1.90371C2.7612 1.58645 3.47468 1.45214 4.18443 1.51569C4.89418 1.57923 5.57245 1.83815 6.144 2.26371C6.71564 1.83798 7.39407 1.57897 8.10399 1.51542C8.81391 1.45188 9.52755 1.58628 10.1657 1.90371C10.8023 2.21935 11.3382 2.70635 11.7131 3.30993C12.0881 3.91352 12.2872 4.60974 12.288 5.32029ZM18.8794 13.8831C16.0526 13.8831 13.7597 16.1649 13.7597 18.978C13.7597 21.7911 16.0526 24.0711 18.8803 24.0711C21.708 24.0711 24 21.7911 24 18.978C24 17.6271 23.46 16.3311 22.5 15.3754C21.5378 14.4197 20.2365 13.8835 18.8803 13.884M18.8803 21.1423C18.4506 21.1429 18.0304 21.0164 17.6726 20.7786C17.3147 20.5408 17.0353 20.2024 16.8694 19.806C16.7048 19.4105 16.6617 18.9749 16.7457 18.5547C16.8297 18.1346 17.0369 17.749 17.3409 17.4471C17.646 17.144 18.0339 16.9377 18.4558 16.8542C18.8777 16.7706 19.3149 16.8135 19.7126 16.9774C20.1095 17.1403 20.449 17.4174 20.6882 17.7736C20.9273 18.1298 21.0552 18.549 21.0557 18.978C21.0541 19.5534 20.8241 20.1047 20.4161 20.5106C20.0082 20.9164 19.4557 21.1437 18.8803 21.1423ZM12.288 17.7043V24.0711H9.34371V17.6726C9.34371 17.3657 9.17914 17.0811 8.91171 16.9277C8.78026 16.8522 8.63131 16.8125 8.47971 16.8125C8.32812 16.8125 8.17917 16.8522 8.04771 16.9277C7.78114 17.082 7.61571 17.3649 7.61571 17.6726V24.0711H4.67229V17.6726C4.67229 17.1977 4.28486 16.8129 3.80829 16.8129C3.33086 16.8129 2.94429 17.1986 2.94429 17.6726V24.0711H0V17.7043C0 16.2574 0.822 14.934 2.12314 14.2869C2.76128 13.9697 3.4748 13.8355 4.18455 13.8992C4.8943 13.9629 5.57253 14.222 6.144 14.6477C6.71564 14.222 7.39407 13.963 8.10399 13.8994C8.81391 13.8359 9.52755 13.9703 10.1657 14.2877C10.8023 14.6034 11.3382 15.0904 11.7131 15.6939C12.0881 16.2975 12.2872 16.9937 12.288 17.7043Z"
                  fill="white"
                />
              </g>
              <defs>
                <clippath
                  id="clip0_2468_86"
                >
                  <rect
                    fill="white"
                    height="24"
                    transform="translate(0 0.5)"
                    width="24"
                  />
                </clippath>
              </defs>
            </symbol>
            <use
              href="#0f406a85"
            />
          </svg>
        </div>
        Momo me
      </button>
    </div>
  </body>,
  "container": <div>
    <button
      class="center text-sm-rps transition-300 focus-shadow ring-border h-10 rounded-2 bg-[#D82D8B] px-4 py-2.5 text-white-500 ring-[#C5297E] hover:bg-[#C1177C] md:h-11 w-full"
    >
      <div
        class="mr-2 w-5 scale-[83.275%] md:w-6 md:scale-100"
      >
        <svg
          viewBox="0 0 24 24"
        >
          <symbol
            fill="none"
            id="0f406a85"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              clip-path="url(#clip0_2468_86)"
            >
              <path
                d="M18.8803 1.5C16.0517 1.5 13.7597 3.78 13.7597 6.594C13.7597 9.40714 16.0526 11.6871 18.8803 11.6871C21.708 11.6871 24 9.40714 24 6.594C24 5.24229 23.46 3.94714 22.5 2.99143C21.5378 2.03565 20.2365 1.49947 18.8803 1.5ZM18.8803 8.75829C18.5953 8.75907 18.3129 8.70372 18.0493 8.59538C17.7857 8.48704 17.546 8.32784 17.3439 8.12687C17.1418 7.9259 16.9813 7.68709 16.8715 7.42408C16.7617 7.16107 16.7048 6.87901 16.704 6.594C16.7047 6.30892 16.7615 6.02677 16.8713 5.76366C16.981 5.50055 17.1415 5.26164 17.3436 5.06058C17.5457 4.85951 17.7854 4.70024 18.0491 4.59184C18.3128 4.48345 18.5952 4.42807 18.8803 4.42886C19.1653 4.42818 19.4476 4.48365 19.7112 4.59209C19.9748 4.70053 20.2144 4.85983 20.4164 5.06088C20.6184 5.26193 20.7789 5.50081 20.8885 5.76386C20.9982 6.02691 21.055 6.30899 21.0557 6.594C21.0555 6.8784 20.9991 7.15995 20.8899 7.42251C20.7806 7.68507 20.6205 7.92346 20.4189 8.124C20.01 8.53043 19.4568 8.75848 18.8803 8.75829ZM12.288 5.32029V11.6871H9.34371V5.28857C9.34371 4.98171 9.17914 4.69714 8.91171 4.54371C8.78026 4.46822 8.63131 4.42849 8.47971 4.42849C8.32812 4.42849 8.17917 4.46822 8.04771 4.54371C7.78114 4.698 7.61571 4.98086 7.61571 5.28857V11.6871H4.67229V5.28857C4.67229 4.81371 4.28486 4.42886 3.80829 4.42886C3.33086 4.42886 2.94429 4.81371 2.94429 5.28857V11.6871H0V5.32029C0 3.87343 0.822 2.55086 2.12314 1.90371C2.7612 1.58645 3.47468 1.45214 4.18443 1.51569C4.89418 1.57923 5.57245 1.83815 6.144 2.26371C6.71564 1.83798 7.39407 1.57897 8.10399 1.51542C8.81391 1.45188 9.52755 1.58628 10.1657 1.90371C10.8023 2.21935 11.3382 2.70635 11.7131 3.30993C12.0881 3.91352 12.2872 4.60974 12.288 5.32029ZM18.8794 13.8831C16.0526 13.8831 13.7597 16.1649 13.7597 18.978C13.7597 21.7911 16.0526 24.0711 18.8803 24.0711C21.708 24.0711 24 21.7911 24 18.978C24 17.6271 23.46 16.3311 22.5 15.3754C21.5378 14.4197 20.2365 13.8835 18.8803 13.884M18.8803 21.1423C18.4506 21.1429 18.0304 21.0164 17.6726 20.7786C17.3147 20.5408 17.0353 20.2024 16.8694 19.806C16.7048 19.4105 16.6617 18.9749 16.7457 18.5547C16.8297 18.1346 17.0369 17.749 17.3409 17.4471C17.646 17.144 18.0339 16.9377 18.4558 16.8542C18.8777 16.7706 19.3149 16.8135 19.7126 16.9774C20.1095 17.1403 20.449 17.4174 20.6882 17.7736C20.9273 18.1298 21.0552 18.549 21.0557 18.978C21.0541 19.5534 20.8241 20.1047 20.4161 20.5106C20.0082 20.9164 19.4557 21.1437 18.8803 21.1423ZM12.288 17.7043V24.0711H9.34371V17.6726C9.34371 17.3657 9.17914 17.0811 8.91171 16.9277C8.78026 16.8522 8.63131 16.8125 8.47971 16.8125C8.32812 16.8125 8.17917 16.8522 8.04771 16.9277C7.78114 17.082 7.61571 17.3649 7.61571 17.6726V24.0711H4.67229V17.6726C4.67229 17.1977 4.28486 16.8129 3.80829 16.8129C3.33086 16.8129 2.94429 17.1986 2.94429 17.6726V24.0711H0V17.7043C0 16.2574 0.822 14.934 2.12314 14.2869C2.76128 13.9697 3.4748 13.8355 4.18455 13.8992C4.8943 13.9629 5.57253 14.222 6.144 14.6477C6.71564 14.222 7.39407 13.963 8.10399 13.8994C8.81391 13.8359 9.52755 13.9703 10.1657 14.2877C10.8023 14.6034 11.3382 15.0904 11.7131 15.6939C12.0881 16.2975 12.2872 16.9937 12.288 17.7043Z"
                fill="white"
              />
            </g>
            <defs>
              <clippath
                id="clip0_2468_86"
              >
                <rect
                  fill="white"
                  height="24"
                  transform="translate(0 0.5)"
                  width="24"
                />
              </clippath>
            </defs>
          </symbol>
          <use
            href="#0f406a85"
          />
        </svg>
      </div>
      Momo me
    </button>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`components button momo > renders correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <button
        class="center text-sm-rps transition-300 focus-shadow ring-border h-10 rounded-2 bg-[#D82D8B] px-4 py-2.5 text-white-500 ring-[#C5297E] hover:bg-[#C1177C] md:h-11"
      >
        <div
          class="mr-2 w-5 scale-[83.275%] md:w-6 md:scale-100"
        >
          <svg
            viewBox="0 0 24 24"
          >
            <symbol
              fill="none"
              id="0f406a85"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                clip-path="url(#clip0_2468_86)"
              >
                <path
                  d="M18.8803 1.5C16.0517 1.5 13.7597 3.78 13.7597 6.594C13.7597 9.40714 16.0526 11.6871 18.8803 11.6871C21.708 11.6871 24 9.40714 24 6.594C24 5.24229 23.46 3.94714 22.5 2.99143C21.5378 2.03565 20.2365 1.49947 18.8803 1.5ZM18.8803 8.75829C18.5953 8.75907 18.3129 8.70372 18.0493 8.59538C17.7857 8.48704 17.546 8.32784 17.3439 8.12687C17.1418 7.9259 16.9813 7.68709 16.8715 7.42408C16.7617 7.16107 16.7048 6.87901 16.704 6.594C16.7047 6.30892 16.7615 6.02677 16.8713 5.76366C16.981 5.50055 17.1415 5.26164 17.3436 5.06058C17.5457 4.85951 17.7854 4.70024 18.0491 4.59184C18.3128 4.48345 18.5952 4.42807 18.8803 4.42886C19.1653 4.42818 19.4476 4.48365 19.7112 4.59209C19.9748 4.70053 20.2144 4.85983 20.4164 5.06088C20.6184 5.26193 20.7789 5.50081 20.8885 5.76386C20.9982 6.02691 21.055 6.30899 21.0557 6.594C21.0555 6.8784 20.9991 7.15995 20.8899 7.42251C20.7806 7.68507 20.6205 7.92346 20.4189 8.124C20.01 8.53043 19.4568 8.75848 18.8803 8.75829ZM12.288 5.32029V11.6871H9.34371V5.28857C9.34371 4.98171 9.17914 4.69714 8.91171 4.54371C8.78026 4.46822 8.63131 4.42849 8.47971 4.42849C8.32812 4.42849 8.17917 4.46822 8.04771 4.54371C7.78114 4.698 7.61571 4.98086 7.61571 5.28857V11.6871H4.67229V5.28857C4.67229 4.81371 4.28486 4.42886 3.80829 4.42886C3.33086 4.42886 2.94429 4.81371 2.94429 5.28857V11.6871H0V5.32029C0 3.87343 0.822 2.55086 2.12314 1.90371C2.7612 1.58645 3.47468 1.45214 4.18443 1.51569C4.89418 1.57923 5.57245 1.83815 6.144 2.26371C6.71564 1.83798 7.39407 1.57897 8.10399 1.51542C8.81391 1.45188 9.52755 1.58628 10.1657 1.90371C10.8023 2.21935 11.3382 2.70635 11.7131 3.30993C12.0881 3.91352 12.2872 4.60974 12.288 5.32029ZM18.8794 13.8831C16.0526 13.8831 13.7597 16.1649 13.7597 18.978C13.7597 21.7911 16.0526 24.0711 18.8803 24.0711C21.708 24.0711 24 21.7911 24 18.978C24 17.6271 23.46 16.3311 22.5 15.3754C21.5378 14.4197 20.2365 13.8835 18.8803 13.884M18.8803 21.1423C18.4506 21.1429 18.0304 21.0164 17.6726 20.7786C17.3147 20.5408 17.0353 20.2024 16.8694 19.806C16.7048 19.4105 16.6617 18.9749 16.7457 18.5547C16.8297 18.1346 17.0369 17.749 17.3409 17.4471C17.646 17.144 18.0339 16.9377 18.4558 16.8542C18.8777 16.7706 19.3149 16.8135 19.7126 16.9774C20.1095 17.1403 20.449 17.4174 20.6882 17.7736C20.9273 18.1298 21.0552 18.549 21.0557 18.978C21.0541 19.5534 20.8241 20.1047 20.4161 20.5106C20.0082 20.9164 19.4557 21.1437 18.8803 21.1423ZM12.288 17.7043V24.0711H9.34371V17.6726C9.34371 17.3657 9.17914 17.0811 8.91171 16.9277C8.78026 16.8522 8.63131 16.8125 8.47971 16.8125C8.32812 16.8125 8.17917 16.8522 8.04771 16.9277C7.78114 17.082 7.61571 17.3649 7.61571 17.6726V24.0711H4.67229V17.6726C4.67229 17.1977 4.28486 16.8129 3.80829 16.8129C3.33086 16.8129 2.94429 17.1986 2.94429 17.6726V24.0711H0V17.7043C0 16.2574 0.822 14.934 2.12314 14.2869C2.76128 13.9697 3.4748 13.8355 4.18455 13.8992C4.8943 13.9629 5.57253 14.222 6.144 14.6477C6.71564 14.222 7.39407 13.963 8.10399 13.8994C8.81391 13.8359 9.52755 13.9703 10.1657 14.2877C10.8023 14.6034 11.3382 15.0904 11.7131 15.6939C12.0881 16.2975 12.2872 16.9937 12.288 17.7043Z"
                  fill="white"
                />
              </g>
              <defs>
                <clippath
                  id="clip0_2468_86"
                >
                  <rect
                    fill="white"
                    height="24"
                    transform="translate(0 0.5)"
                    width="24"
                  />
                </clippath>
              </defs>
            </symbol>
            <use
              href="#0f406a85"
            />
          </svg>
        </div>
        Momo me
      </button>
    </div>
  </body>,
  "container": <div>
    <button
      class="center text-sm-rps transition-300 focus-shadow ring-border h-10 rounded-2 bg-[#D82D8B] px-4 py-2.5 text-white-500 ring-[#C5297E] hover:bg-[#C1177C] md:h-11"
    >
      <div
        class="mr-2 w-5 scale-[83.275%] md:w-6 md:scale-100"
      >
        <svg
          viewBox="0 0 24 24"
        >
          <symbol
            fill="none"
            id="0f406a85"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              clip-path="url(#clip0_2468_86)"
            >
              <path
                d="M18.8803 1.5C16.0517 1.5 13.7597 3.78 13.7597 6.594C13.7597 9.40714 16.0526 11.6871 18.8803 11.6871C21.708 11.6871 24 9.40714 24 6.594C24 5.24229 23.46 3.94714 22.5 2.99143C21.5378 2.03565 20.2365 1.49947 18.8803 1.5ZM18.8803 8.75829C18.5953 8.75907 18.3129 8.70372 18.0493 8.59538C17.7857 8.48704 17.546 8.32784 17.3439 8.12687C17.1418 7.9259 16.9813 7.68709 16.8715 7.42408C16.7617 7.16107 16.7048 6.87901 16.704 6.594C16.7047 6.30892 16.7615 6.02677 16.8713 5.76366C16.981 5.50055 17.1415 5.26164 17.3436 5.06058C17.5457 4.85951 17.7854 4.70024 18.0491 4.59184C18.3128 4.48345 18.5952 4.42807 18.8803 4.42886C19.1653 4.42818 19.4476 4.48365 19.7112 4.59209C19.9748 4.70053 20.2144 4.85983 20.4164 5.06088C20.6184 5.26193 20.7789 5.50081 20.8885 5.76386C20.9982 6.02691 21.055 6.30899 21.0557 6.594C21.0555 6.8784 20.9991 7.15995 20.8899 7.42251C20.7806 7.68507 20.6205 7.92346 20.4189 8.124C20.01 8.53043 19.4568 8.75848 18.8803 8.75829ZM12.288 5.32029V11.6871H9.34371V5.28857C9.34371 4.98171 9.17914 4.69714 8.91171 4.54371C8.78026 4.46822 8.63131 4.42849 8.47971 4.42849C8.32812 4.42849 8.17917 4.46822 8.04771 4.54371C7.78114 4.698 7.61571 4.98086 7.61571 5.28857V11.6871H4.67229V5.28857C4.67229 4.81371 4.28486 4.42886 3.80829 4.42886C3.33086 4.42886 2.94429 4.81371 2.94429 5.28857V11.6871H0V5.32029C0 3.87343 0.822 2.55086 2.12314 1.90371C2.7612 1.58645 3.47468 1.45214 4.18443 1.51569C4.89418 1.57923 5.57245 1.83815 6.144 2.26371C6.71564 1.83798 7.39407 1.57897 8.10399 1.51542C8.81391 1.45188 9.52755 1.58628 10.1657 1.90371C10.8023 2.21935 11.3382 2.70635 11.7131 3.30993C12.0881 3.91352 12.2872 4.60974 12.288 5.32029ZM18.8794 13.8831C16.0526 13.8831 13.7597 16.1649 13.7597 18.978C13.7597 21.7911 16.0526 24.0711 18.8803 24.0711C21.708 24.0711 24 21.7911 24 18.978C24 17.6271 23.46 16.3311 22.5 15.3754C21.5378 14.4197 20.2365 13.8835 18.8803 13.884M18.8803 21.1423C18.4506 21.1429 18.0304 21.0164 17.6726 20.7786C17.3147 20.5408 17.0353 20.2024 16.8694 19.806C16.7048 19.4105 16.6617 18.9749 16.7457 18.5547C16.8297 18.1346 17.0369 17.749 17.3409 17.4471C17.646 17.144 18.0339 16.9377 18.4558 16.8542C18.8777 16.7706 19.3149 16.8135 19.7126 16.9774C20.1095 17.1403 20.449 17.4174 20.6882 17.7736C20.9273 18.1298 21.0552 18.549 21.0557 18.978C21.0541 19.5534 20.8241 20.1047 20.4161 20.5106C20.0082 20.9164 19.4557 21.1437 18.8803 21.1423ZM12.288 17.7043V24.0711H9.34371V17.6726C9.34371 17.3657 9.17914 17.0811 8.91171 16.9277C8.78026 16.8522 8.63131 16.8125 8.47971 16.8125C8.32812 16.8125 8.17917 16.8522 8.04771 16.9277C7.78114 17.082 7.61571 17.3649 7.61571 17.6726V24.0711H4.67229V17.6726C4.67229 17.1977 4.28486 16.8129 3.80829 16.8129C3.33086 16.8129 2.94429 17.1986 2.94429 17.6726V24.0711H0V17.7043C0 16.2574 0.822 14.934 2.12314 14.2869C2.76128 13.9697 3.4748 13.8355 4.18455 13.8992C4.8943 13.9629 5.57253 14.222 6.144 14.6477C6.71564 14.222 7.39407 13.963 8.10399 13.8994C8.81391 13.8359 9.52755 13.9703 10.1657 14.2877C10.8023 14.6034 11.3382 15.0904 11.7131 15.6939C12.0881 16.2975 12.2872 16.9937 12.288 17.7043Z"
                fill="white"
              />
            </g>
            <defs>
              <clippath
                id="clip0_2468_86"
              >
                <rect
                  fill="white"
                  height="24"
                  transform="translate(0 0.5)"
                  width="24"
                />
              </clippath>
            </defs>
          </symbol>
          <use
            href="#0f406a85"
          />
        </svg>
      </div>
      Momo me
    </button>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
